Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -129,6 +129,21 @@ public final class ManagePledgeViewModel:
       .takeWhen(cancelPledgeSelected)
       // swiftlint:disable:next line_length
       .map { _ in Strings.We_dont_allow_cancelations_that_will_cause_a_project_to_fall_short_of_its_goal_within_the_last_24_hours() }
+
+    let managePledgeMenuType: Signal<Koala.ManagePledgeMenuCTAType, Never> = Signal.merge(
+      self.menuOptionSelectedSignal.filter { $0 == .updatePledge }.mapConst(.updatePledge),
+      self.menuOptionSelectedSignal.filter { $0 == .changePaymentMethod }.mapConst(.changePaymentMethod),
+      self.menuOptionSelectedSignal.filter { $0 == .chooseAnotherReward }.mapConst(.chooseAnotherReward),
+      self.menuOptionSelectedSignal.filter { $0 == .contactCreator }.mapConst(.contactCreator),
+      self.menuOptionSelectedSignal.filter { $0 == .cancelPledge }.mapConst(.cancelPledge)
```

Comment:
Here it's simpler if you create a function that takes a ManagePledgeAlertAction and returns a Koala.ManagePledgeMenuCTAType, using a switch statement.

This would simplify a lot this code by just using:

```
let managePledgeMenuType: Signal<Koala.ManagePledgeMenuCTAType, Never> = self.menuOptionSelectedSignal
      .map(managePledgeMenuCTAType(for:))
```